tests/components/mqtt/test_fan.py

```diff
@@ -499,37 +526,41 @@ async def test_controlling_state_via_topic_and_json_message_shared_topic(
 
     state = hass.states.get("fan.test")
     assert state.state == STATE_UNKNOWN
+    assert state.attributes.get("direction") is None
     assert not state.attributes.get(ATTR_ASSUMED_STATE)
 
     async_fire_mqtt_message(
         hass,
         "shared-state-topic",
-        '{"state":"ON","preset_mode":"eco","oscillation":"oscillate_on","percentage": 50}',
+        '{"state":"ON","preset_mode":"eco","oscillation":"oscillate_on","percentage": 50, "direction": "forward"}',
```

Comment:
Can we split this string to avoid long lines and make ik more readable?

Reply:
Fixed with a multiline string (added in a separate commit).